app/controllers/bills_controller.rb

```diff
@@ -77,6 +81,13 @@ def destroy
     @bill.destroy
     redirect_to admins_path
   end
+  
+  def common
+    @countOfProcedure =Bill.group(:procedure_id).count
```

Comment:
use snake case naming convention for variable names.
```suggestion
    @count_of_procedure = Bill.group(:procedure_id).count
```